phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
The previous default was this:

# Bandpass filter low corner frequency
filter_low = 500.

# Bandpass filter high corner frequency
filter_high = 0.95 * .5 * sample_rate

And the units on both of these are Hz (1/sec). Neither are in samples or
seconds! The second one is computed from the sample_rate (which also has
units of Hz) to be just below Nyquist, that's all. In fact I guess this
filter_high part is filtering almost nothing out - that's really the
low-pass filter you'd need on your hardware before the signal was
digitized. You might as well just be using a high-pass at this point rather
than bandpass.

While we're here, is there a reason that you don't get to specify the
filter that's used to make the LFP [low.kwd]? This should probably be a
parameter too.

Nick

On Tue, Jun 30, 2015 at 6:04 PM, Cyrille Rossant <notifications@github.com>
wrote:

> In phy/cluster/algorithms.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33596857>:
>
> > @@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
> >      def _create_filter(self):
> >          rate = self._kwargs['sample_rate']
> >          low = self._kwargs['filter_low']
>
> well I'd say these two parameters are less similar than what their names
> imply, and they play different roles... but @shabnamkadir
> <https://github.com/shabnamkadir> will know more.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33596857>.
>